projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/side-bar.component.html

```diff
@@ -33,9 +33,20 @@
     </div>
     <div class="outlineItem" *ngIf="selectedView === 'bookmark'">
       <ng-container *ngIf="(bookmarks$ | async) as bookmarks">
-        <a *ngFor="let bookmark of bookmarks"
-           class="outlineItem" (click)="goToBookmark(bookmark)">{{ bookmark.name }}</a>
+        <div *ngFor="let bookmark of bookmarks">
+          <div class="outlineItem" *ngIf="bookmark.id!==editableBookmark; else inputBookmark">
+            <a (click)="goToBookmark(bookmark)">
+              {{ bookmark.name }}
+            </a>
+            <button class="bookmark__rename" (click)="editBookmark(bookmark.id)" ></button>
+            <button class="bookmark__delete" (click)="deleteBookmark(bookmark)"></button>
+          </div>
+          <ng-template #inputBookmark>
+            <input #bookmarkName class="bookmark__input" placeholder="{{bookmark.name}}" [maxLength]="BOOKMARK_CHAR_LIMIT">
```

Comment:
why is the bookmark.name a placeholder rather than the value, that stops the user from being able to edit it, and the only option he has is to replace it completely

Reply:
Yes this is true.. I will change it :)